homeassistant/components/cast/media_player.py

```diff
@@ -392,15 +471,27 @@ def async_cast_discovered(discover: ChromecastInfo):
         self._available = True
         self.cast_status = chromecast.status
         self.media_status = chromecast.media_controller.status
-        _LOGGER.debug("Connection successful!")
+        _LOGGER.debug("[%s %s (%s:%s)] Connection successful!",
+                      self.entity_id, self._cast_info.friendly_name,
+                      self._cast_info.host, self._cast_info.port)
         self.async_schedule_update_ha_state()
 
+    async def async_del_cast_info(self, cast_info):
+        """Remove the service."""
+        self.services.discard(cast_info.service)
```

Comment:
Remove the service name from the set of services, the pychromecast object will still poll the set every 5s even if it goes empty.